### DISCUSSION TITLE
Links back to parent directory does not work in Markdown

### DISCUSSION THREAD

**Discussion by fluentcodes** [community/community#13849] [Other Feature Feedback, Questions, & Ideas]
I would like to expand a sandbox concept to something I called "GitSamples". It uses Topic Branches and README.md files in folders for documentation as Repo on Github. I've got a problem when navigate from a README.md in one folder back to another with [..](..). Back navigation is simple not possible in Markdowns.
The only solution I've found is /../../tree/BRANCH/PATH. So all the documention only work in that specific branch. Renaming or cloning would destroy all links in documentation. Is there another solution? 
Or is there something planned? Relative folder links works fine unless you want to go back.
Best 
Werner

My investigations: 
https://github.com/fluentcodes/GitSamples-GIT/tree/github-readme-links

**Comment by alex-cognizant** [comment]
Hi @fluentcodes ,

I've arrived here because I can't get the relative paths to work with README.md file at all.

I saw that the same issue happens in other Github repos that were supposed to work before. Example:
- [Stackoverflow answer](https://stackoverflow.com/a/72457556)
- [Github example repo](https://github.com/mark-anders/relative-image-url)

It's strange, as Github itself is even recommending using relative paths:
[Relative links and image paths in readme files](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files)

Did you get more information about this? I tried to search for official Github issue ticket or something, but I didn't find anything :(


Thanks!

**Reply by futuresea-dev** [reply]
Thank you

**Comment by sancarn** [comment]
@fluentcodes Not sure if you're aware but `dir/subdir` no longer works. It 404s
